cross_list_courses/migrations/0001_initial.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql = "INSERT INTO lti_permissions_ltipermission "
+                  "(permission, school_id, canvas_role, allow) "
+                  "VALUES ('cross_listing', '*', 'AccountAdmin', TRUE),"
+                  "('cross_listing', '*', 'Account Admin', TRUE);",
+            reverse_sql='delete  from lti_permissions_ltipermission where permission="cross_listing"',
+        ),
+    ]
+
```

Comment:
After I run this locally, I get the following message:
```
(canvas_account_admin_tools)vagrant@vagrant-ubuntu-trusty-64:~/tlt/canvas_account_admin_tools(task/sapnamysore/TLT-2180/Cross-list-courses) $ python manage.py migrate
Operations to perform:
  Synchronize unmigrated apps: django_auth_lti, sslserver, course_info, staticfiles, debug_toolbar, messages, proxy, djangular, icommons_ui
  Apply all migrations: cross_list_courses, sessions, canvas_account_admin_tools, auth, icommons_common, contenttypes, lti_permissions
Synchronizing apps without migrations:
  Creating tables...
    Running deferred SQL...
  Installing custom SQL...
Running migrations:
  Rendering model states... DONE
  Applying cross_list_courses.0001_initial... OK
The following content types are stale and need to be deleted:

    icommons_common | siscoursedata

Any objects related to these content types by a foreign key will also
be deleted. Are you sure you want to delete these content types?
If you're unsure, answer 'no'.

    Type 'yes' to continue, or 'no' to cancel:
```
Anyone have a lead on what this is and how to resolve it?